eacl/target.go

```diff
@@ -29,12 +29,18 @@ func ecdsaKeysToPtrs(keys []ecdsa.PublicKey) []*ecdsa.PublicKey {
 
 // BinaryKeys returns list of public keys to identify
 // target subject in a binary format.
+//
+// The resulting slice of bytes is a serialized compressed public key. See [elliptic.MarshalCompressed].
```

Comment:
`Each element of the resulting slice is a`...

Reply:
Updated